site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,68 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* FusionAuth wasn't awesome enough, so we added a robust Threat Detection feature for enterprise customers. This feature includes: 
```

Comment:
I havent pulled this down, but check bullets for consistency IE - (* , ** , *** , ****) / leveling